build.ps1

```diff
@@ -159,7 +162,17 @@ function RunTests {
 }
 
 function UnitTests {
-    Invoke-Execute { RunTests -Filter "*.Tests" }
+    Invoke-Execute { RunTests -Filter "*.Tests.Unit" }
+}
+
+function RunE2E {
+    Invoke-Execute { RunTests -Filter "*.Tests.E2E" }
+}
+
+function E2ETests {
+    Invoke-Step { DockerBuild }
```

Comment:
The e2e tests require a local docker image, so the script will do both the build and then run the tests